README.md

```diff
@@ -199,6 +199,55 @@ An error will be raised:
 
 `ActiveModel::ValidationError: Validation failed: Title can't be blank`
 
+#### Content Areas
+
+A component can be configured to have multiple 'content areas' which can be captured and later rendered in the component's view. For example:
+
+`app/components/modal_component.rb`:
+```ruby
+class ModalComponent < ActionView::Component::Base
+  validates :user, :header, :body, presence: true
+
+  with_content_areas :header
```

Comment:
Do we want to use `body` or the built-in `content` in this example? I'd probably lean towards `body`:

```suggestion
  with_content_areas :header, :body
```